File: clients/csharp/src/MustelaLog.Client.Core/Models/TimeBucket.cs
namespace MustelaLog.Client.Core.Models;

/// <summary>Einfacher Zeit-Bucket für den clientseitigen Zeitverlauf.</summary>
public sealed class TimeBucket
{
    public string Label { get; set; } = string.Empty;
    public int Count { get; set; }
}

File: clients/java/src/main/java/de/sasd/mustelalog/client/model/TimeBucket.java
package de.sasd.mustelalog.client.model;

/**
 * Simple time series bucket for the current result set.
 */
public final class TimeBucket
{
    private final String label;
    private final long count;

    public TimeBucket(String label, long count)
    {
        this.label = label;
        this.count = count;
    }

    public String getLabel() { return label; }
    public long getCount() { return count; }
}
